### PR TITLE
SHARE-330 prod docker image

### PR DESCRIPTION
**Pull request by marweb1996**
---
### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroweb-Symfony/blob/develop/.github/contributing.md) and [wiki pages](https://github.com/Catrobat/catroweb-Symfony/wiki/) of this repository.

- [ ] Include the name and id of the Jira ticket in the PR’s title eg.: `SHARE-666 The devils ticket`
- [ ] Choose the proper base branch (*develop*)
- [ ] Confirm that the changes follow the project’s coding guidelines
- [ ] Verify that the changes generate no warnings and errors 
- [ ] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Verify that all tests are passing (CI), if not please state the test cases in the [section](#Tests) below
- [ ] Perform a self-review of the changes
- [ ] Stick to the project’s git workflow (rebase and squash your commits)
- [ ] Verify that your changes do not have any conflicts with the base branch
- [ ] Put your ticket into the `Code Review` section in [Jira](https://jira.catrob.at/)
- [ ] Post a message in the *#catroweb* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer
- [ ] Check that your pull request has been successfully deployed to https://web-test-1.catrob.at/

### Additional Description
`TODO: Add additional information that is not in your commit-message here`

### Tests - additional information
`TODO: add additional information about testruns here`

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,16 +1,24 @@
+# ---------------------------------------------- Build Time Arguments --------------------------------------------------
+ARG PHP_VERSION="7.4"
+ARG DEBIAN_VERSION="buster"
+ARG NODE_VERSION="latest"
```
Comment (dmetzner) [inline docker/Dockerfile]: "latest" hat jetzt schon mehrmals zu Problemen geführt. Die sollten wir besser locken.
Im Moment wär das version 15 bei Node
bei composer müsste es noch gecheckt werden

**Comment by dmetzner** [comment]
Die anderen configs müsste man dann wohl auch anpassen, wie php.ini in production. Sollten in Puppet/Confuence dokumentiert sein

File: docker/Dockerfile (hunk 1 above)
Comment (marweb1996) [reply docker/Dockerfile]: Ja klar das ist auch nicht wirklich geeignet da latest nicht zwingend vorhanden sein muss. Wurde bis jetzt nur mal zum Testen verwendet.

**Comment by marweb1996** [comment]
Ja klar die restlichen configs werden alle noch angepasst.

**Pull request closed without merging**